doc/sphinx/credits.rst

```diff
@@ -1404,3 +1404,137 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
 | Santiago de Chile, March 2018,
 | Matthieu Sozeau for the |Coq| development team
 |
+
+Credits: version 8.9
+--------------------
+
+|Coq| version 8.9 contains the result of refinements and stabilization
+of features and deprecations or removals of deprecated features,
+cleanups of the internals of the system and API along with a few new
+features. This release includes many user-visible changes, including
+deprecations that are documented in ``CHANGES.md``.
+
+TODO:
+We only list the most important ones:
+or
+The main user visible changes are:
+
+- Kernel: mutually defined records are now supported, by P.M. Pédrot.
+
+- Notations: support for autonomous grammars of terms called "custom
+  entries", by H. Herbelin (see chapter "Syntax extensions" of the
+  reference manual).
+
+- Notations: deprecated notations will be removed in the next version of
+  |Coq|, see the``CHANGES.md`` file for a script to ease porting,
+  by J. Gross.
+
+- Tactics: Introduction tactics `intro`/`intros` on a goal that is an
```

Comment:
I am not fluent with reStructuredText, but shouldn't those be double backquotes? (And similarly below.)

Reply:
In CHANGES.md they're simple. @Zimmi48 ?

Reply:
You were right, for literal code `` `` is the right syntax